script/brave_license_helper.py

```diff
@@ -60,7 +60,7 @@ def AddBraveCredits(root, prune_paths, special_cases, prune_dirs,
             "URL": "https://github.com/brave/brave-extension",
             "License": "MPL-2.0",
         },
-        os.path.join('brave', 'vendor', 'challenge_bypass_ristretto_ffi'): {
+        os.path.join('brave', 'third_party', 'challenge_bypass_ristretto'): {
```

Comment:
If we're moving this into `brave-core` and it's not in an external repo anymore, then what's needed is to:
1. delete this block
2. add an exception for this path just above line 23

Reply:
Done.